src/StudentTest.java

```diff
@@ -0,0 +1,11 @@
+public class StudentTest {
+    public static void main(String[] arg) {
+        Student student1 = new Student("Julia", "Leng", 5,55,100,90,90);
+        Student student2 = new Student("Max", "Doling", 4,56,80,100,95);
+        Student student3 = new Student("Helen", "Kittr", 2,57,90,79,88);
+        System.out.println(student1.firstName + " average Score = " + (student1.mathScore+ student1.economicScore+ student1.foreignLanguageScore)/3);
```

Comment:
To avoid duplication to find the average marks for students it's needed to create method for it and use the method:
 void avgMarkAllSubjects() {
    double result = (mathMark + economicMark + foreignLangMark) / 3;
    System.out.println("The average mark for all 3 subjects for " + lastName + " is " + result);
  }